kayak_render_macros/src/lib.rs

```diff
@@ -154,8 +154,21 @@ pub fn dyn_partial_eq(_: TokenStream, input: TokenStream) -> TokenStream {
 #[proc_macro]
 pub fn use_state(initial_state: TokenStream) -> TokenStream {
     let initial_state = parse_macro_input!(initial_state as syn::Expr);
+    let found_crate = proc_macro_crate::crate_name("kayak_core");
+    let kayak_core = if let Ok(found_crate) = found_crate {
+        match found_crate {
+            proc_macro_crate::FoundCrate::Itself => quote! { crate },
+            proc_macro_crate::FoundCrate::Name(name) => {
+                let ident = syn::Ident::new(&name, proc_macro2::Span::call_site());
+                quote!(#ident)
+            }
+        }
+    } else {
+        quote!(kayak_ui::core)
+    };
+
     let result = quote! {{
-        use kayak_core::{Bound, MutableBound};
+        use #kayak_core::{Bound, MutableBound};
```

Comment:
Ah, didn't know I needed to do this. We'll also need to update `use_effect.rs` as well then:
https://github.com/StarArawn/kayak_ui/blob/13458f0806e9df499968b13db21966fb76f40592/kayak_render_macros/src/use_effect.rs#L58